app/src/main/java/ch/admin/bag/dp3t/util/StringUtil.kt

```diff
@@ -163,6 +163,15 @@ object StringUtil {
 		}
 	}
 
+	fun getFormattedWeekdayWithDate(timestamp: Long, context: Context): String {
```

Comment:
very confusing, that this method has the same name as the one in DateUtils, but does something different